readme.md

```diff
@@ -0,0 +1,182 @@
+# Robotic Vision Integration Package (RVIP) Documentation
+![Build Status](https://img.shields.io/gitlab/pipeline/cardboardcode/rvip)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![codecov](https://codecov.io/gl/cardboardcode/rvip_ros/branch/master/graph/badge.svg?token=9XFJvJLbpK)](https://codecov.io/gl/cardboardcode/rvip)
```

Comment:
Same here.

Reply:
For **code coverage badge**, since there is no official ros-industrial/rvip codecov link up yet, will probably only update this after there is one.